generator/typesystem.h

```diff
@@ -58,6 +61,27 @@
     };
 #endif
 
+/* Global endl (::endl) is used extensively in the generator .cpp files.  This
+ * was supported by Qt until Qt6.  In Qt5.14 Qt::endl was added in anticipation
+ * of the Qt6 change and the use of global endl could be avoided (it does not
+ * seem to have been explicitly deprecated).  This gives backward compatibility
+ * for global endl in Qt6 (not Qt5, where global endl was still available).
+ *
+ * Note that 'constexpr' is available in Qt6 because Qt6 requires C++17;
+ * consexpr was introduced in C++11.  Likewise for decltype.  Qt::endl is a
```

Comment:
Nitpicking - typo constexpr

Reply:
Typo?  It seems correct to me; it's spelled the same in the code and the code compiles!

Reply:
Oh... sorry; the second one...